src/handlePermissions.js

```diff
@@ -11,17 +11,22 @@ export const requestPermissions = async (hasVideoAndAudio, CameraManager, permis
             return isAuthorized;
         }
     } else if (Platform.OS === 'android') {
-        const grantedCamera = await PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.CAMERA, {
-            title: permissionDialogTitle,
-            message: permissionDialogMessage,
-          });
+        const cameraPermissionDialog = permissionDialogTitle || permissionDialogMessage ? {
```

Comment:
no need for this, as we have defaultProps

this will always exist

Reply:
Great, so if change the default props to...

```js
permissionDialogTitle: null,
permissionDialogMessage: null,
microphonePermissionDialogTitle: null,
microphonePermissionDialogMessage: null,
```

What information do you want expanding in the docs? The docs don’t currently mention the default prop types so I just want to clarify what you think would be good to expand/edit on?

Reply:
if you don't provide a permissionDialogTitle it won't show any message, just the default OS message

Reply:
Oh strange... I kept getting blank dialog boxes. I will re-test it and get back to you.